### PR TITLE
[Merged by Bors] - Update Postgresql Helm chart version

### PR DESCRIPTION
**Pull request by razvan**
:green_circle: CI: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/druid-operator-it-custom/63/

### CODE REVIEW

File: tests/templates/kuttl/authorizer/03-install-druid.yaml.j2 [hunk 1]
```diff
@@ -11,8 +11,8 @@ metadata:
   name: derby-druid
 spec:
   image:
-    productVersion: "{{ test_scenario['values']['druid'].split('-stackable')[0] }}"
-    stackableVersion: "{{ test_scenario['values']['druid'].split('-stackable')[1] }}"
+    productVersion: "{{ test_scenario['values']['druid-latest'].split('-stackable')[0] }}"
+    stackableVersion: "{{ test_scenario['values']['druid-latest'].split('-stackable')[1] }}"
```
Comment (maltesander) [inline tests/templates/kuttl/authorizer/03-install-druid.yaml.j2:15]: This currently saves one test. The druid-opa-authorizer version is coupled to respective druid versions, i would keep the test for all versions.

**Review by maltesander — APPROVED** [review]
Thanks! LGTM if integration tests pass.

File: tests/templates/kuttl/authorizer/03-install-druid.yaml.j2 (hunk 1 above)
Comment (razvan) [reply tests/templates/kuttl/authorizer/03-install-druid.yaml.j2]: ok, reverted.

**Comment by razvan** [comment]
bors merge

**Pull request closed without merging**